unreal_asset/src/asset/name_map.rs

```diff
@@ -91,6 +91,11 @@ impl NameMap {
         &self.name_map_index_list[index as usize]
     }
 
+    /// Get a name reference by an FName map index as a `String`
+    pub fn get_owned_name_reference(&self, index: i32) -> String {
```

Comment:
There is not really an owned reference. This could just be called `get_owned_name`.

Reply:
lol true